aspose/src/test/java/com/testautomationguru/utility/ImageUtil.java

```diff
@@ -7,19 +7,19 @@
 
 import javax.imageio.ImageIO;
 
-class ImageUtil {
+public class ImageUtil {
 
 	static Logger logger = Logger.getLogger(ImageUtil.class.getName());
 
-	static boolean compareAndHighlight(final BufferedImage img1, final BufferedImage img2, String fileName, boolean highlight, int colorCode) throws IOException {
-
+	static boolean compareAndHighlight(final BufferedImage img1, final BufferedImage img2, String fileName, boolean highlight, int colorCode, double deviation) throws IOException {
```

Comment:
Ook hier misschien nog `deviation` hernoemen naar `allowedDeviation` of `allowedDifference` of zoiets?

Reply:
PS: Op mijn laptop zijn sommige images identiek, zo niet dan zie ik diffs van 0.12, 0.39, en eenmaal de hoogste is 0.49.

Allemaal binnen de grens van 0.51 dus.